self-tests.yml

```diff
@@ -317,6 +317,15 @@ functions:
       params:
         files:
           - "gopath/src/github.com/evergreen-ci/evergreen/bin/jstests/*.xml"
+  remove-test-results:
```

Comment:
why can't this just be a third command in attach test results?

Reply:
Attach test results is also called in post. It handles more than just lint.